db_connection.py

```diff
@@ -27,25 +32,25 @@ def connect_db(self, connection_parameters):
                     port=connection_parameters['port']
                 )
                 self.cursor = self.conn.cursor()
-                print("Connection Success!")
+                logger.info("Connection Success!")
                 return 
             except OperationalError as e:
                 attempts += 1
-                print(f"Connection error: {e}. Attempt {attempts}/{MAX_CONNECTION_ATTEMPTS}.")
+                logger.error(f"Attempt {attempts}/{MAX_CONNECTION_ATTEMPTS}. Connection error: {e}")
                 time.sleep(DELAY)
             except Exception as ex:
-                print(f'[Err] exception has been caught during establishing connection to db: {ex}')
+                logger.critical(f'[Err] exception has been caught during establishing connection to db: {ex}')
```

Comment:
[Err] is not mandatory, because logger marks level of logging record.

Reply:
resolved